namui-cli/src/services/bundle/mod.rs

```diff
@@ -0,0 +1,213 @@
+mod lexicon;
+mod token;
+
+use super::resource_collect_service::CollectOperation;
+use crate::util::get_cli_root_path;
+use lexicon::*;
+use std::{collections::HashMap, path::PathBuf};
+use token::*;
+
+#[derive(Debug)]
+pub struct NamuiBundleManifest {
```

Comment:
I moved manifest from util to services and it will serve functions to other services.